app/models/attached_file.rb

```diff
@@ -1,8 +1,14 @@
-# typed: strict
+# typed: false
 # frozen_string_literal: true
 
 # Models a File that is attached to a Work
 class AttachedFile < ApplicationRecord
   belongs_to :work
   has_one_attached :file
+
+  def blob
```

Comment:
We use the AF instance as a proxy for the blob, and the changes to this file make it easier to display richer information about each AF.

Reply:
What if there is no attachment here?  Should we handle that case?

Reply:
@jcoyne that seems like an error condition, in which case we might want to raise an exception. no?